bin/travis/test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# pwd is the git repo.
+set -e
+
+# For XVFB Selenium tests.
+export DISPLAY=:99.0
+
+python manage.py test \
+  --noinput --logging-clear-handlers
```

Comment:
We should add nosenicedots to `requirements/dev.txt` and then add `--with-nicedots`. It produces much better output.